scripts/getDependencies.xml

```diff
@@ -30,15 +30,8 @@
 				<property name="jtregTar" value="jtreg_5_1_b01"/>
 			</then>
 			<elseif>
-				<!-- version 11 -->
-				<matches pattern="^11$" string="${JDK_VERSION}"/>
-				<then>
-					<property name="jtregTar" value="jtreg_6_1"/>
-				</then>
-			</elseif>
-			<elseif>
-				<!-- versions 17, 21, 22, 23 -->
-				<matches pattern="^(17|2[123])$" string="${JDK_VERSION}"/>
+				<!-- versions 11, 17, 21, 22, 23 -->
+				<matches pattern="^(11|17|2[123])$" string="${JDK_VERSION}"/>
```

Comment:
Why do we need to check versions and fail, requiring an update every time there is a new jdk version? Could it just load 7.3.1.1 until we find a newer jdk version requires something different? The OpenJDK testing checks the version and fails if it's not recent enough.

Reply:
I see no need to check the version either. I will update the code